src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapTheNewGeneration.java

```diff
@@ -0,0 +1,214 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.elemnt.JavascriptActions;
+import com.wikia.webdriver.common.core.url.Page;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.AutoplayVuap;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.VuapAssertions;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.VuapVideos;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class TestAdsVuapTheNewGeneration extends TemplateNoFirstLoad {
+
+  private static final long MAX_AUTOPLAY_MOVIE_DURATION = 40L;
+
+  private static final String FANDOM_ARTICLE_WESTWORLD_LINK = "http://adeng.fandom.wikia.com/articles/whats-coming-westworld-finale";
+
+  private static final String PROJECT43_TNG_ARTICLE_LINK = "http://project43.wikia.com/wiki/DevTemplates/VUAP/TNG";
+
+  private AdsBaseObject openPageWithVideoInLocalStorage(Page page, String player, String video) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "openPageWithVideoInLocalStorage" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)